src/lib/Scenes/Home/Components/Sales/Components/SaleListItem.tsx

```diff
@@ -79,8 +79,10 @@ export class SaleListItem extends React.Component<RelayProps, null> {
     return (screenSize.width - gutterSize) / numColumns
   }
 
-  handleTap(navigateToUrl) {
-    Switchboard.presentNavigationViewController(this, navigateToUrl)
+  handleTap = () => {
+    const { sale: { live_url_if_open, href } } = this.props
+    const url = (live_url_if_open || href) as string
+    Switchboard.presentNavigationViewController(this, url)
```

Comment:
@alloy - This change follows [this comment](https://github.com/artsy/collector-experience/issues/743#issuecomment-348724297) as well as this [Metaphysics PR](https://github.com/artsy/metaphysics/pull/849) but I don't think this is correct for what the switchboard expects. Can you provide some insight as to what the best approach would be?

Reply:
It's for the Eigen SwitchBoard, ( here's [the live auctions code](https://github.com/artsy/eigen/blob/master/Artsy/App/ARSwitchBoard.m#L226-L248)) - I don't think the Emission switchboard has support for it

Reply:
When I pass in the full url to `live(-staging).artsy.net` nothing crashes but the the screen remains blank

Reply:
The switchboard [is trivial in Emission](https://github.com/artsy/emission/blob/master/Example/Emission/AppDelegate.m#L216-L250) it could just not support routing to urls that aren't force. Eigen should be able to handle it, you can follow the control flow in Xcode to figure out why in that file if you're interested.